source/index.html.md

```diff
@@ -523,8 +573,41 @@ payment method must be omitted.
     <i>Optional</i> <br />
     A reference to an external object, such as an order number.
   </dd>
+  <dt>series[type]</dt>
+  <dd>
+    (installment|recurring|unscheduled|cardholder) <br />
+    <i>Optional. Cannot be present if <code>series[previous]</code> is present.</i><br />
+    Indicate if the series of transactions is <code>installment</code>,
+    <code>recurring</code>, <code>unscheduled</code> credential on file
+    (where subsequent transactions are initiated by the merchant), or
+    <code>cardholder</code>-initiated credential on file (where all transactions
+    in series is cardholder-initiated).
+    <br />
+    If provided, it indicates that the transaction is first in series of the
+    given type initiated by the cardholder.
+  </dd>
+  <dt>series[previous]</dt>
+  <dd>
+    reference
+    <br />
+    <i>Optional. Cannot be present if <code>series[type]</code> is present.</i>
+    <br />
+    A reference to the previous approved authorization in the series (use
```

Comment:
"the previous approved authorization" is confusing, as there might be more than one.

Maybe "one of the previous, approved authorizations"? Or do we only want the latest or the first?